aten/src/ATen/native/ts_native_functions.yaml

```diff
@@ -141,42 +141,65 @@ full_codegen:
   - upsample_nearest2d_backward
   - zero.functional
   - narrow_copy.SymInt
+  - alias_copy
```

Comment:
i guess you went through and added support for the superset of supportable ops, even if they weren't yet supported?  Is there test coverage for these?  I assume for ops like `split_copy` you attempted to add it but then it failed something so leaving it for later?

Reply:
Yep - I'll update with a quick comment on that. I tried to add them originally, but it looked like the LTC codegen didn't support ops with `TensorList` outputs.